cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -146,11 +167,25 @@ data LiveViewState a = LiveViewState
     , lvsMempoolCapacity      :: !Word64
     , lvsMempoolCapacityBytes :: !Word64
     , lvsMessage              :: !(Maybe a)
+    -- Async threads.
     , lvsUIThread             :: !LiveViewThread
     , lvsMetricsThread        :: !LiveViewThread
     , lvsNodeThread           :: !LiveViewThread
+
+    , lvsNodeKernel           :: Maybe (LVNodeKernel blk)
```

Comment:
Shouldn't this be strict?

Reply:
Careful that simply adding a `!` on a `Maybe` is pretty useless.

Reply:
I would agree - except - given that this data structure is also an instance of  `NoUnexpectedThunks` you might need it?

Reply:
Done, by adding `SMaybe`.